content/rancher/v2.x/en/operation/ingress/_index.md

```diff
@@ -5,3 +5,62 @@ draft: true
 ---
 
 ---
+
+**Introduction**
+
+Access to every service deployed on k8s, is provided by ingress controller, a l7 load balancer. Ingress controller used by default on Rancher is nginx. It provides and secure (if tls available) access to services by named virtual host or configuring default site to a service. 
+
+By default, services are provided and secured by ingress controller, but some other scenarios could be configured. External lb's (hardware or software) could be used to provide access to services. 
+
+**Scenarios**
+
+Depending of the architecture and where tls termination is configured, 3 distinct configuration could be used to provide access to services.
+
+`ingress` 
+
+- No external lb's used. Optional external lb l4 could be used.. 
+- Entrypoints and tls termination are configured at ingress controller. 
```

Comment:
L7 routing rules (e.g. based on `host`, `path`) and corresponding endpoints are configured at the ingress controller. TLS termination can be configured either at the ingress controller or an optional external l4 LB.

Reply:
There is a specific section for using external l4 lb. TLS couldn't be configured at external l4 lb.